streams/src/main/java/org/apache/kafka/streams/kstream/internals/Change.java

```diff
@@ -42,12 +48,13 @@ public boolean equals(final Object o) {
             return false;
         }
         final Change<?> change = (Change<?>) o;
-        return Objects.equals(newValue, change.newValue) &&
-                Objects.equals(oldValue, change.oldValue);
+        return Objects.equals(newValue, change.newValue)
+            && Objects.equals(oldValue, change.oldValue)
+            && isLatest == change.isLatest;
```

Comment:
I'm on the fence about updating the equals method here. I'd like for it to be updated in this way but because serialization does not depend on this new boolean, it could be the case that round-trip serialization causes equals() to no longer evaluate to true, which seems confusing. 

Maybe it doesn't matter if the equals method isn't called anywhere (I couldn't find any usages). Curious to hear reviewer opinions.

Reply:
Yeah. Let's leave it as-is. -- Thanks for explaining.